src/pages/settings/Security/TwoFactorAuth/CopyCodesPage.tsx

```diff
@@ -60,7 +60,7 @@ function CopyCodesPage({route}: TwoFactorAuthPageProps) {
         }
         toggleTwoFactorAuth(true);
         // eslint-disable-next-line react-compiler/react-compiler, react-hooks/exhaustive-deps -- We want to run this when component mounts
-    }, [isUserValidated, accountMetadata]);
+    }, [isUserValidated, accountMetadata?.status]);
```

Comment:
`accountMetadata` is not nullable, so we can remove the optional chain here:

```suggestion
    }, [isUserValidated, accountMetadata.status]);
```

Reply:
@roryabraham Updates PR